frontend/src/features/projects/list/SPLProjectListView.test.tsx

```diff
@@ -0,0 +1,174 @@
+import { SPLProjectListView } from './SPLProjectListView';
+import React from 'react';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+import { render, cleanup, wait } from '@testing-library/react';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { ILookupCode } from 'actions/lookupActions';
+import * as API from 'constants/API';
+import { Provider } from 'react-redux';
+import * as reducerTypes from 'constants/reducerTypes';
+import service from '../apiService';
+import { noop } from 'lodash';
+import { Formik } from 'formik';
+
+const testData = {
```

Comment:
There may already be test data kept in a central location.  If there isn't might be a nice chance to make it.  Then other tests could reference this mocked data.

Reply:
Sounds like a good idea, but is there already a central location with test data that I can use, if not do we need to implement this in this ticket?

Reply:
You don't need to implement in this ticket.  Just feedback on ways we might be able to improve.  If there is a central location for keeping this stuff already then it would make sense to use it though.